tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/iso-639-3-macrolanguages.tab

```diff
@@ -1,454 +1,455 @@
-lah	xhe	A
-lav	ltg	A
-lav	lvs	A
-luy	bxk	A
-luy	ida	A
-luy	lkb	A
-luy	lko	A
-luy	lks	A
-luy	lri	A
-luy	lrm	A
-luy	lsm	A
-luy	lto	A
-luy	lts	A
-luy	lwg	A
-luy	nle	A
-luy	nyd	A
-luy	rag	A
-man	emk	A
-man	mku	A
-man	mlq	A
-man	mnk	A
-man	msc	A
-man	mwk	A
-man	myq	R
-mlg	bhr	A
-mlg	bjq	R
-mlg	bmm	A
-mlg	bzc	A
-mlg	msh	A
-mlg	plt	A
-mlg	skg	A
-mlg	tdx	A
-mlg	tkg	A
-mlg	txy	A
-mlg	xmv	A
-mlg	xmw	A
-mon	khk	A
-mon	mvf	A
-msa	bjn	A
-msa	btj	A
-msa	bve	A
-msa	bvu	A
-msa	coa	A
-msa	dup	A
-msa	hji	A
-msa	ind	A
-msa	jak	A
-msa	jax	A
-msa	kvb	A
-msa	kvr	A
-msa	kxd	A
-msa	lce	A
-msa	lcf	A
-msa	liw	A
-msa	max	A
-msa	meo	A
-msa	mfa	A
-msa	mfb	A
-msa	min	A
-msa	mly	R
-msa	mqg	A
-msa	msi	A
-msa	mui	A
-msa	orn	A
-msa	ors	A
-msa	pel	A
-msa	pse	A
-msa	tmw	A
-msa	urk	A
-msa	vkk	A
-msa	vkt	A
-msa	xmm	A
-msa	zlm	A
-msa	zmi	A
-msa	zsm	A
-mwr	dhd	A
-mwr	mtr	A
-mwr	mve	A
-mwr	rwr	A
-mwr	swv	A
-mwr	wry	A
-nep	dty	A
-nep	npi	A
-nor	nno	A
-nor	nob	A
-oji	ciw	A
-oji	ojb	A
-oji	ojc	A
-oji	ojg	A
-oji	ojs	A
-oji	ojw	A
-oji	otw	A
-ori	ory	A
-ori	spv	A
-orm	gax	A
-orm	gaz	A
-orm	hae	A
-orm	orc	A
-pus	pbt	A
-pus	pbu	A
-pus	pst	A
-que	cqu	R
-que	qub	A
-que	qud	A
-que	quf	A
-que	qug	A
-que	quh	A
-que	quk	A
-que	qul	A
-que	qup	A
-que	qur	A
-que	qus	A
-que	quw	A
-que	qux	A
-que	quy	A
-que	quz	A
-que	qva	A
-que	qvc	A
-que	qve	A
-que	qvh	A
-que	qvi	A
-que	qvj	A
-que	qvl	A
-que	qvm	A
-que	qvn	A
-que	qvo	A
-que	qvp	A
-que	qvs	A
-que	qvw	A
-que	qvz	A
-que	qwa	A
-que	qwc	A
-que	qwh	A
-que	qws	A
-que	qxa	A
-que	qxc	A
-que	qxh	A
-que	qxl	A
-que	qxn	A
-que	qxo	A
-que	qxp	A
-que	qxr	A
-que	qxt	A
-que	qxu	A
-que	qxw	A
-raj	bgq	A
-raj	gda	A
-raj	gju	A
-raj	hoj	A
-raj	mup	A
-raj	wbr	A
-rom	rmc	A
-rom	rmf	A
-rom	rml	A
-rom	rmn	A
-rom	rmo	A
-rom	rmw	A
-rom	rmy	A
-sqi	aae	A
-sqi	aat	A
-sqi	aln	A
-sqi	als	A
-srd	sdc	A
-srd	sdn	A
-srd	src	A
-srd	sro	A
-swa	swc	A
-swa	swh	A
-syr	aii	A
-syr	cld	A
-tmh	taq	A
-tmh	thv	A
-tmh	thz	A
-tmh	ttq	A
-uzb	uzn	A
-uzb	uzs	A
-yid	ydd	A
-yid	yih	A
-zap	zaa	A
-zap	zab	A
-zap	zac	A
-zap	zad	A
-zap	zae	A
-zap	zaf	A
-zap	zai	A
-zap	zam	A
-zap	zao	A
-zap	zaq	A
-zap	zar	A
-zap	zas	A
-zap	zat	A
-zap	zav	A
-zap	zaw	A
-zap	zax	A
-zap	zca	A
-zap	zoo	A
-zap	zpa	A
-zap	zpb	A
-zap	zpc	A
-zap	zpd	A
-zap	zpe	A
-zap	zpf	A
-zap	zpg	A
-zap	zph	A
-zap	zpi	A
-zap	zpj	A
-zap	zpk	A
-zap	zpl	A
-zap	zpm	A
-zap	zpn	A
-zap	zpo	A
-zap	zpp	A
-zap	zpq	A
-zap	zpr	A
-zap	zps	A
-zap	zpt	A
-zap	zpu	A
-zap	zpv	A
-zap	zpw	A
-zap	zpx	A
-zap	zpy	A
-zap	zpz	A
-zap	zsr	A
-zap	ztc	R
-zap	zte	A
-zap	ztg	A
-zap	ztl	A
-zap	ztm	A
-zap	ztn	A
-zap	ztp	A
-zap	ztq	A
-zap	zts	A
-zap	ztt	A
-zap	ztu	A
-zap	ztx	A
-zap	zty	A
-zha	ccx	R
-zha	ccy	R
-zha	zch	A
-zha	zeh	A
-zha	zgb	A
-zha	zgm	A
-zha	zgn	A
-zha	zhd	A
-zha	zhn	A
-zha	zlj	A
-zha	zln	A
-zha	zlq	A
-zha	zqe	A
-zha	zyb	A
-zha	zyg	A
-zha	zyj	A
-zha	zyn	A
-zha	zzj	A
-zho	cdo	A
-zho	cjy	A
-zho	cmn	A
-zho	cnp	A
-zho	cpx	A
-zho	csp	A
-zho	czh	A
-zho	czo	A
-zho	gan	A
-zho	hak	A
-zho	hsn	A
-zho	lzh	A
-zho	mnp	A
-zho	nan	A
-zho	wuu	A
-zho	yue	A
-zza	diq	A
+M_Id	I_Id	I_Status
```

Comment:
Why does it look in GitHub like the whole file changed?
Is there an inadvertent whitespace change on every line?

Same question for all of the .tab files.

Reply:
@markusicu because it was checked in with CRLF.  I'm updating it to be consistent with all other source files.